Lib/crypt.py

```diff
@@ -79,7 +79,12 @@ def _add_method(name, *args, rounds=None):
     method = _Method(name, *args)
     globals()['METHOD_' + name] = method
     salt = mksalt(method, rounds=rounds)
-    result = crypt('', salt)
+    result = None
+    try:
+        result = crypt('', salt)
+    except OSError:
```

Comment:
Can you check that the `errno` of this error is `EINVAL`?

I think it would also be good to ensure that at least one method works.

Reply:
Good thinking! Done! Thanks a lot :+1: